nginx-controller/nginx/config.go

```diff
@@ -46,6 +46,8 @@ type Config struct {
 
 	Ports    []int
 	SSLPorts []int
+
+	EndpointsInfo map[string]EndpointInfo
```

Comment:
We don't really need to have this field in the `Config` type. We pass `EndpointsInfo` in `IngressEx` to the `generateNginxCfg` method of `Configurator` and generate maps in that method. There is no need to store Endpoints info in the `Config`.